ci-operator/jobs/openshift/installer/openshift-installer-main-presubmits.yaml

```diff
@@ -8639,6 +8639,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-vsphere-ovn-multi-network,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^main$
+    - ^main-
+    cluster: vsphere02
+    context: ci/prow/e2e-vsphere-ovn-multi-network-techpreview
+    decorate: true
+    labels:
+      ci-operator.openshift.io/cloud: vsphere
+      ci-operator.openshift.io/cloud-cluster-profile: vsphere-elastic
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-main-e2e-vsphere-ovn-multi-network-techpreview
+    optional: true
```

Comment:
We later on might need to skip the job for doc changes only. There have been some [efforts](https://github.com/openshift/release/pull/62800) to reduce these runs :D

```
skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)
```